server/controllers/feed.controller.js

```diff
@@ -14,8 +14,17 @@ function list(req, res, next) {
   .exec()
 
   .then((feed) => {
-    if (!feed) { return res.json([]);}
-    res.json(feed.feedItems);
+    // If user is new and doesn't have a feed, use Jeff's feed
+    if (!feed) {
+      Feed.findOne({user: '597a06d7f0dc67003db0c4c0'})
+      .exec()
+      .then((jeffsFeed) => {
+        res.json(jeffsFeed.feedItems)
```

Comment:
If jeff doesn't exist for some reason and there is no feedItems we can return empty [].